pkgs/development/tools/gertty/default.nix

```diff
@@ -0,0 +1,48 @@
+{ python3, python3Packages, stdenv }:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "gertty";
+  version = "1.6.0";
+
+  nativeBuildInputs = with python3Packages; [
```

Comment:
It's unlikely that these packages are needed in the nativeBuildInputs. You can find in `15.17.2.2.1.1. buildPythonPackage parameters` (Somewhere below here: https://nixos.org/nixpkgs/manual/#reference ) that in the case of python:

nativeBuildInputs ? []: Build-time only dependencies. Typically executables as well as the items listed in setup_requires.

buildInputs ? []: Build and/or run-time dependencies that need to be be compiled for the host machine. Typically non-Python libraries which are being linked.

checkInputs ? []: Dependencies needed for running the checkPhase. These are added to nativeBuildInputs when doCheck = true. Items listed in tests_require go here.

propagatedBuildInputs ? []: Aside from propagating dependencies, buildPythonPackage also injects code into and wraps executables with the paths included in this list. Items listed in install_requires go here.